docs/_toc.yml

```
@@ -35,6 +35,11 @@ subtrees:
     entries:
       - file: contributing/contributing.md
 
+  - caption: Add-Ons
+    entries:
+      - url: https://github.com/LucaRocco/liqo_resources_dashboard
+        title: Peering Dashboard
```

```suggestion
        title: Resources Dashboard
```

Would make it consistent with the readme

---

Agreed with Marco.
My proposal would be to call it "Liqo public peering dashboard".